docs/tutorials/blog.md

```diff
@@ -203,12 +203,8 @@ type Post = {
   title: string;
 };
 
-type LoaderData = {
-  posts: Array<Post>;
-};
-
 export const loader = async () => {
-  return json<LoaderData>({
+  return json<{ posts: Array<Post>; }>({
```

Comment:
Unless you want to deliberately assert at this point what you expect the loader to return, you don't need this, `useLoaderData` will still be able to infer the type.
```suggestion
  return json({
```
I set the type only when I have something more complex, like disjoint union.